controllers/etcd_controller.go

```diff
@@ -1206,6 +1218,10 @@ func (r *EtcdReconciler) getMapFromEtcd(etcd *druidv1alpha1.Etcd) (map[string]in
 		etcdValues["etcdDefragTimeout"] = etcd.Spec.Etcd.EtcdDefragTimeout
 	}
 
+	if etcd.Spec.Etcd.MemberHeartbeat != nil {
+		etcdValues["heartbeatDuration"] = etcd.Spec.Etcd.MemberHeartbeat
```

Comment:
Nit: Could we have matching names here, e.g. either both "heartbeatDuration" or both "memberHeartbeat" (or "memberHeartbeatDuration")?